Ghidra/Features/Base/src/main/java/ghidra/app/util/opinion/OmfLoader.java

```diff
@@ -478,6 +539,8 @@ private void processExternalSymbols(OmfFileHeader header, Program program, TaskM
 		SymbolTable symbolTable = program.getSymbolTable();
 		Language language = program.getLanguage();
 
+		ArrayList<OmfSymbolRecord> publicSymbols = header.getPublicSymbols();
```

Comment:
Map<String, OmfSymbol> publicSymbols = header.getPublicSymbols()
				.stream()
				.flatMap(symbolRec -> symbolRec.getSymbols().stream())
				.collect(Collectors.toMap(sym -> sym.getName(), Function.identity()));

// in OmfSymbolRecord:
	public List<OmfSymbol> getSymbols() {
		return List.of(symbol);
	}